website/docs/dashboard/dashboard_user_guide.md

```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
+This is the start page on the dashboard.
+It will appear when you first turn on your Amiga.
+Here is where you can change the display's language.
+You may choose between English, Spanish, and French.
+Pressing the START button will take you to the home screen.
 
-Alternatively if you receive the "Check pendant calibration" page,
-this means that either you have defective pendant or your pendant needs to be calibrated.
+## Home Screen
 
-<img src="https://user-images.githubusercontent.com/64480560/233126475-5416e312-9d77-4552-876c-f4129d30f244.jpg"
-     width="400"
-     height="500" />
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/954a66d5-2155-48f1-9830-25428374fa2a"/>
 
-:::tip
-This button will take you to the same page as the `pendant` section of the Settings tab.
-:::
+This is the home screen of the dashboard. In the center sits the spedometer,
+ where you can switch between metric and standard units and adjust to the travel speed of your Amiga.
 
-If your pendant is not being detected at all and if it says `No Pendant Detected`
-please contact the Farm-ng Support immediately by visiting [our Support page](https://amiga.farm-ng.com/docs/support/).
+git  On the right side of the screen you will see several icons displaying:
 
-## Home Screen
+- Battery Level
+- Average motor temperature
+- Connection to Pendant
+- Preview of each motor's health
 
-<img src="https://user-images.githubusercontent.com/64480560/233131301-372ac33f-1a21-4aa9-999e-5093eaa53360.jpg"
-     width="400"
-     height="500" />
+You can always come back to this screen by pressing the upper left icon on the dashboard.
 
-This is the home screen of the dashboard. On this screen you will find:
+## Motor Status Screen
 
-- The speedometer
-- A preview of motor health
-- Average motor temperature
-- Battery level of the amiga
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/d0cc17f2-3cdf-49c5-ab02-e412cbec56bd"/>
 
-## Motor Status Screen
+In this screen you can see the control state of your Amiga:
 
-<img src="https://user-images.githubusercontent.com/64480560/232093032-cd99e871-5f09-41bb-9485-6e3cf1c3871b.jpg"
-     width="330"
-     height="300" />
+- Active
+- Cruise
+- Auto Ready/Active
+- E-Stopped
 
-<img src="https://user-images.githubusercontent.com/64480560/232093025-48b6792e-08a3-48f8-b83e-52d4813d84e9.jpg"
-     width="330"
-     height="300" />
+ You will also see details on each motor's:
 
-This screen for monitoring each motor in much more detail than that of the home screen.
-On this screen you can:
+- Voltage
+- RPM
+- Temperature
+- Current draw for each motor/controller
 
-- Monitor the control state of the Amiga
-- Voltage, RPM, temperature, and current of each motor / motor controller
-- State of the CAN bus
+Lastly, CANbus states appear on the right side of the display.
+Follow this link for an indepth description of your Amiga's
+[Canbus Service](https://amiga.farm-ng.com/docs/concepts/canbus_service/)
```

Comment:
That link is for the brain interacting with the can bus. It's not really applicable to the CAN states IMO 

```suggestion
```